receiver/k8sclusterreceiver/internal/deployment/deployments.go

```diff
@@ -5,7 +5,7 @@ package deployment // import "github.com/open-telemetry/opentelemetry-collector-
 
 import (
 	"go.opentelemetry.io/collector/pdata/pcommon"
-	conventions "go.opentelemetry.io/otel/semconv/v1.6.1"
```

Comment:
Actually just realised that `k8scluster` receiver is also partially updated here. I guess that's not intentional, right?

I'd expect this component to have significant breaking changes in general :)

Reply:
i reviewed all the keys from 1.6.1 -> 1.37.0 for the keys included in the files in this PR and didn't find any changes.

Reply:
i dont have a strong preference, but i would rather we have less references to old semconv wherever possible